docs/mpcie_modules.md: Description of ASM1061 module (#148)
pcengines/apu2-documentation: docs/mpcie_modules.md

    sudo apt-get install firmware-realtek
    ```

## SATA controllers

> 

Reviewer: ```suggestion
```
One new line is enough

### ASMedia ASM1061

ASM1061 was tested with Delock 95233 mPCIe 2xSATA module.

![ASM1061](images/delock_95233.png)

**Compatible slots**

Due to apu boards construction, it is possible to test that module only under

> apu2 mPCIe2 slot. Slot mPCIe1 at any platform is unreachable. SATA connectors in

Reviewer: ```suggestion
apu2 mPCIe2 slot. Capacitors placed near the mPCIe1 slot prevent connecting the SATA cables to the module. SATA connectors in
```

Delock 95233 are on the side, which makes it impossible to attach hard drive
(capacitors next to mPCIe1 slot block it).

> 

Reviewer: I would type here that the module is not supported for now.

The test configurations and possible reasons section should land to the docs/debug directory of this repository.

When You create such file, please link it in this document, so anyone who reads this will check why the module is not supported.

Author: Move it to docs/debug/mpcie2.md and created 2 sections there: Tests results and Possible reasons.

  warnings and errors occurred. Example log is shown below.

  ```
  [   68.190693] ata3: exception Emask 0x10 SAct 0x0 SErr 0x4040000 action 0xe frozen
  [   68.198142] ata3: irq_stat 0x00000040, connection status changed
  [   68.204199] ata3: SError: { CommWake DevExch }
  [   68.208743] ata3: hard resetting link
  [   73.826743] ata3: SATA link up 3.0 Gbps (SStatus 123 SControl 300)
  [   73.839192] ata3.00: ATA-8: ST500LM012 HN-M500MBB, 2AR10002, max UDMA/133
  [   73.846011] ata3.00: 976773168 sectors, multi 0: LBA48 NCQ (depth 31/32), AA
  [   73.859579] ata3.00: configured for UDMA/133
  [   73.863903] ata3: EH complete
  [   73.867515] scsi 2:0:0:0: Direct-Access     ATA      ST500LM012 HN-M5 0002 PQ: 0 ANSI: 5
  [   73.876569] sd 2:0:0:0: Attached scsi generic sg1 type 0
  [   73.876592] sd 2:0:0:0: [sdb] 976773168 512-byte logical blocks: (500 GB/466 GiB)
  [   73.876597] sd 2:0:0:0: [sdb] 4096-byte physical blocks
  [   73.876665] sd 2:0:0:0: [sdb] Write Protect is off
  [   73.876827] sd 2:0:0:0: [sdb] Write cache: enabled, read cache: enabled, doesn't support DPO or FUA
  [   73.941291]  sdb: sdb1 sdb2 < sdb5 >

>   [   73.946498] sd 2:0:0:0: [sdb] Attached SCSI disk

Reviewer: It looks like the device is enumerated properly. The first errors seem to be early errors due to how SATA disks are treated (they should be connected from the start). The disk must be spinned first, that is why it reports frozen probably etc. Have You tried mounting any sdb partition @piotr-kleins ?

Author: Can't mount anything because system hangs after that logs and gives no response at all

    sudo apt-get install firmware-realtek
    ```

## SATA controllers

### ASMedia ASM1061

> ASM1061 was tested with Delock 95233 mPCIe 2xSATA module.

Reviewer: Add single sentence that module is not supported. Capital letters would be good @piotr-kleins

Author: Done

![ASM1061](images/delock_95233.png)

**Compatible slots**

Due to apu boards construction, it is possible to test that module only under
apu2 mPCIe2 slot. Capacitors placed near the mPCIe1 slot prevent connecting the

> SATA cables to the module. Therefor, module was tested only in mPCIe2 slot.

Reviewer: ```suggestion
SATA cables to the module. Therefore, module was tested only in mPCIe2 slot.
```

Reviewer: @piotr-kleins the typo left

Author: Weird, in changed files it is fixed. In my local repo also

Reviewer: Ok there is another typo in another doc
